lib/rails/dom/testing/assertions/selector_assertions.rb

```diff
@@ -263,6 +263,25 @@ def assert_dom_email(&block)
         end
         alias_method :assert_select_email, :assert_dom_email
 
+
+        # Changes the depth of the document_root_element for the operations
+        # nested within the provided block. Accepts a singular Nokogiri node or
+        # an Nokogiri NodeSet as its argument.
+        #
+        #  within document_root_element.at_css("main") do
```

Comment:
Can people also just supply a CSS selector string directly?

Reply:
Looks like they can’t, but we should just support that.

Reply:
Done!